source/fuzz/transformation_vector_shuffle.cpp

```diff
@@ -56,12 +57,22 @@ bool TransformationVectorShuffle::IsApplicable(
   if (!vector1_instruction || !vector1_instruction->type_id()) {
     return false;
   }
+  // |vector1| can't be an irrelevant id.
+  if (transformation_context.GetFactManager()->IdIsIrrelevant(
+          message_.vector1())) {
+    return false;
+  }
   // The second vector must be an instruction with a type id
   auto vector2_instruction =
       ir_context->get_def_use_mgr()->GetDef(message_.vector2());
   if (!vector2_instruction || !vector2_instruction->type_id()) {
     return false;
   }
+  // |vector2| can't be an irrelevant id.
+  if (transformation_context.GetFactManager()->IdIsIrrelevant(
+          message_.vector2())) {
+    return false;
+  }
```

Comment:
Again, we *could* allow irrelevant ids and just not add facts when they wouldn't be valid.

Reply:
Fixed in 35f5730.